bubblewrap.c

```diff
@@ -1974,6 +1993,15 @@ main (int    argc,
     {
       /* Parent, outside sandbox, privileged (initially) */
 
+      if (opt_info_fd != -1)
+        {
+          cleanup_free char *output = xasprintf ("{\n    \"child-pid\": %i\n}\n", pid);
+          size_t len = strlen (output);
+          if (write (opt_info_fd, output, len) != len)
+            die_with_error ("Write to info_fd");
+          close (opt_info_fd);
+        }
+
```

Comment:
Why is this needed? In the setuid case this allows you to make root write to an arbirary fd. Most checks on privileges are on open(), so its not necessary super-problematic, but still possibly bad.